controller_manager_msgs/CMakeLists.txt

```
@@ -18,6 +18,7 @@ set(msg_files
   msg/ControllerState.msg
 )
 set(srv_files
+  srv/ListControllerInterfaces.srv
```

Or maybe even better `ListHardwareInterfaces`
```suggestion
  srv/ListResourceInterfaces.srv
```

---

ec56888